database/insertData/03_template_A_featureShowcase.js

```diff
@@ -1102,6 +1102,161 @@ exports.queries = [
+                      elementTypePluginCode: "search"
+                      category: QUESTION
+                      isRequired: false
+                      parameters: {
+                        label: "Lookup ATC codes"
+                        description: "Start typing a drug name"
+                        placeholder: "Search medicines"
+                        icon: "pills"
+                        minCharacters: 3
+                        multiSelect: true
+                        source: {
+                          operator: "graphQL",
+                          children: [
+                            "query GetEntitiesByName($search: String!) {entities (filter: { code: \\"\\" description: $search type: \\"[drug]\\" match: \\"contains\\"},offset: 0,first: 25) {data {        code,     description,        type,        uid  }, totalLength }}",
+                            "https://codes.msupply.foundation:2048/graphql",
+                            [
+                              "search"
+                            ],
+                            {
+                              operator: "objectProperties",
+                              children: [
+                                "search.text"
+                              ]
+                            },
+                            "entities.data"
+                          ]
+                        }
+                        displayFormat: {
+                          title: "\${description}",
+                          description: "ATC code: \${code}"
+                        }
+                        resultFormat: {
+                          title: "\${description}",
+                          description: "\${code}"
+                        }
+                      }
+                    }
+                    {
+                      code: "searchOrg"
+                      index: 11
+                      title: "Search Demo (organisations)"
+                      elementTypePluginCode: "search"
+                      category: QUESTION
+                      isRequired: false
+                      helpText: "This one has no \`displayFormat\` prop, so it falls back to a generic \\"default\\" display"
+                      parameters: {
+                        label: "Search for an organisation"
+                        description: "Please enter the organisation's registration code (min 6 chars)"
+                        placeholder: "Search organisations"
+                        icon: "building"
+                        minCharacters: 6
+                        source: {
+                          operator: "graphQL",
+                          children: [
+                            "query GetOrgs($registration: String!) { organisations(filter: {registration: {startsWithInsensitive: $registration}}) { nodes { name registration } } }",
+                            "",
+                            [
+                              "registration"
```

Comment:
Can we add a search for both "registration" or "company name"? I think it would be useful. Not sure if it's feasible, but I imagine just passing the same `search.text` to 2 different search options on the GraphQL query?

Reply:
Yes, the query would be quite straightforward (just nest it within an OR filter), but this was just trying to demonstrate being able to look up a company without being able to just search freely for names, like @andreievg first described.

Not sure which implementation we will do in the actual template, but let's save figuring out what the specs need to be when we update the actual "joinOrg" template, shall we?